docs/core/introduction.md

```diff
@@ -1,238 +1,84 @@
-* Type inference - [C#](../csharp/fundamentals/types/index.md#specifying-types-in-variable-declarations), [F#](../fsharp/language-reference/type-inference.md), [Visual Basic](../visual-basic/programming-guide/language-features/variables/local-type-inference.md).
-* [Type system](../standard/base-types/common-type-system.md)
-* [Unsafe code](../csharp/language-reference/unsafe-code.md)
+The core libraries expose thousands of types, many of which integrate with and fuel the C# language. For example, C#’s `foreach` enables enumerating arbitrary collections, with pattern-based optimizations that enable collections like `List<T>` to be processed simply and efficiently. Resource management may be left up to garbage collection, but prompt cleanup is possible via `IDisposable` and direct language support in `using`.
 
-## Using .NET
+Support for doing multiple things at the same time is fundamental to practically all workloads, whether it be client applications doing background processing while keeping the UI responsive, services handling many thousands of simultaneous requests, devices responding to a multitude of simultaneous stimuli, or high-powered machines parallelizing the processing of compute-intensive operations. Asynchronous programming support is a first-class feature of the C# programming language, which provides the `async` and `await` keywords that make it easy to write and compose asynchronous operations while still enjoying the full benefits of all the control flow constructs the language has to offer.
 
-.NET apps and libraries are built from source code and a project file, using the [.NET CLI](tools/index.md) or an Integrated Development Environment (IDE) like [Visual Studio](https://visualstudio.microsoft.com/).
+The [type system](../standard/base-types/common-type-system.md) offers significant breadth, catering somewhat equally to safety, descriptiveness, dynamism, and native interop. First and foremost, the type system enables an object-oriented programming model. It includes types, (single base class) inheritance, interfaces (including default method implementations), and virtual method dispatch to provide a sensible behavior for all the type layering that object orientation allows. [Generic types](../standard/generics.md) are a pervasive feature that allow specializing classes to one or more types.
 
-The following example is a minimal .NET app:
+The .NET runtime provides automatic memory management via a garbage collector. For any language, its memory management model is likely its most defining characteristic. This is true for .NET languages. .NET has a self-tuning, tracing GC. It aims to deliver “hands off” operation in the general case while offering configuration options for more extreme workloads. The GC is the result of many years of investment, improving and learning from many kinds of workloads.
```

Comment:
```suggestion
The .NET runtime provides automatic memory management via a garbage collector. For any language, its memory management model is likely its most defining characteristic. This is true for .NET languages. .NET has a self-tuning, tracing GC. It aims to deliver "hands off" operation in the general case while offering configuration options for more extreme workloads. The GC is the result of many years of investment. It has learned and improved from many kinds of workloads.
```

Reply:
This change has an agency problem. The original text is a bit ambiguous on who is doing the learning. The proposed text makes it sound like the GC is doing the learning, which it does not.